interface/tasks.py

```diff
@@ -16,4 +16,5 @@ def process_wiki(repo_id):
         return 'Processing finished'
     except Exception as e:
         print(e)
+        raise
```

Comment:
@birkholz This is a WIP PR, I'm rewriting these parts. Going to remove the blanket exception capture here. This line is left from my dev work.